src/drugs.cpp

```diff
@@ -427,6 +438,10 @@ void apply_drug_modifiers_to_ch(struct char_data *ch) {
           GET_CHA(ch) = 1;
           GET_WIL(ch) /= 2;
           break;
+        case DRUG_DEEPWEED:
+          GET_TARGET_MOD(ch)++;
+          PLR_FLAGS(ch).RemoveBit(PLR_PERCEIVE);
```

Comment:
This will pretty regularly strip perception. I think this function is called once every minute or so, maybe even faster? Worth testing.